containers/arangodb.docker/entrypoint_enterprise.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/sh
+set -e
+
+h=$(echo "$ARANGO_LICENSE_KEY" | sha256sum)
+if [ "$h" != "eb657fa5838f00359dd937a3f2a501f891d4f4b78bd304d9f7cb2b6ed3ab50f4  -" ] ; then
```

Comment:
please change to "EVALUATION:125f16ada6047bd17eeeefa3f011070510b5fbd9d85122afdeca72c380e7ac83"